build/ci/send-to-helix.yml

```
@@ -20,8 +20,34 @@ parameters:
   condition: succeeded()                 # optional -- condition for step to execute; defaults to succeeded()
 
 steps:
+- script: ${{ parameters.MsBuildScript}}
```

I think the indentation here is off.
```suggestion
  - script: ${{ parameters.MsBuildScript}}
```